examples/06-verif-manual/vm-007-plastic_compression_of_a_pipe_assembly.py

```diff
@@ -0,0 +1,487 @@
+mapdl.n(node=204, x=3.8197185, z=10)
+
+# GENERATE NODES TO FORM A THETA DEGREE SLICE
+mapdl.ngen(itime=2, inc=4, node1=201, node2=204, dy=theta)
+
+# INSIDE (STEEL) TUBE
+mapdl.type(3)
+mapdl.secnum(1)
+mapdl.e(203, 201, 205, 207)
+
+# OUTSIDE (ALUMINUM) TUBE
+mapdl.secnum(2)
+mapdl.e(204, 202, 206, 208)
+
+# plot elements
+cpos = [(19.67899462804619, 17.856836088414664, 22.644135378046194),
+        (2.03485925, 0.21270071036846988, 5.0),
+        (0.0, 0.0, 1.0)]
+mapdl.eplot(cpos=cpos)
+
+
+###############################################################################
+# Define Boundary Conditions
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Application of symmetric boundary conditions for simplified model.
+
+mapdl.run("C*** APPLY CONSTRAINTS TO PIPE288 MODEL")
+
+mapdl.d(node=1, lab="ALL")  # FIX ALL DOFS FOR BOTTOM END OF PIPE288
+mapdl.d(node=2, lab="UX", lab2="UY", lab3="ROTX", lab4="ROTY", lab5="ROTZ")  # ALLOW ONLY UZ DOF AT TOP END OF PIPE288 MODEL
+
+mapdl.run("C*** APPLY CONSTRAINTS TO SOLID185 AND SHELL181 MODELS")
+
+mapdl.cp(nset=1, lab="UX", node1=101, node2=111, node3=105, node4=115)  # COUPLE NODES AT BOUNDARY IN RADIAL DIR FOR SOLID185
+mapdl.cpsgen(itime=4, nset1=1)
+
+mapdl.cp(5, lab="UX", node1=201, node2=205, node3=203, node4=20)  # COUPLE NODES AT BOUNDARY IN RADIAL DIR FOR SHELL181
+mapdl.cpsgen(itime=2, nset1=5)
+
+mapdl.cp(7, lab="ROTY", node1=201, node2=205)  # COUPLE NODES AT BOUNDARY IN ROTY DIR FOR SHELL181
+mapdl.cpsgen(itime=4, nset1=7)
+
+mapdl.nsel("S", "NODE", "", 101, 212)  # SELECT ONLY NODES IN SOLID185 AND SHELL181 MODELS
+mapdl.nsel("R", "LOC", "Y", 0)  # SELECT NODES AT THETA = 0 FROM THE SELECTED SET
+mapdl.dsym("SYMM", "Y", 1)  # APPLY SYMMETRY BOUNDARY CONDITIONS
+mapdl.nsel("S", "NODE", "", 101, 212)  # SELECT ONLY NODES IN SOLID185 AND SHELL181 MODELS
+mapdl.nsel("R", "LOC", "Y", theta)  # SELECT NODES AT THETA FROM THE SELECTED SET
+
+mapdl.dsym("SYMM", "Y", 1)  # APPLY SYMMETRY BOUNDARY CONDITIONS
+
+mapdl.nsel("ALL")
+mapdl.nsel("R", "LOC", "Z", 0)  # SELECT ONLY NODES AT Z = 0
+
+mapdl.d("ALL", "UZ", 0)  # CONSTRAIN BOTTOM NODES IN Z DIRECTION
+
+mapdl.nsel("ALL")
+mapdl.finish()
+
+###############################################################################
+# Solve
+# ~~~~~
+# Enter solution mode and solve the system. Print the solver output.
+
+# Start solution presederu
+mapdl.run("/SOLU")
+
+# Define solution function.
+def solution(deflect=None):
+    mapdl.nsel("R", "LOC", "Z", 10)
+    mapdl.d(node="ALL", lab="UZ", value=deflect)
+    mapdl.nsel("ALL")
+    mapdl.solve()
+
+
+# Run each load step to reproduce needed deflection subsequently.
+# Load Step 1
+solution(deflect=defl_ls1)
+
+# Load Step 2
+solution(deflect=defl_ls2)
+
+# Load Step 3
+solution(deflect=defl_ls3)
+mapdl.finish()
+
+
+###############################################################################
+# Post-processing
+# ~~~~~~~~~~~~~~~
+# Enter post-processing.
+
+# Enter the post-processing routine
+mapdl.run("/OUT,")
```

Comment:
Probably don't need to direct output. gRPC will be overriding the output commands anyway.